src/complexity.h

```diff
@@ -56,5 +59,55 @@ struct LeastSq {
 // Function to return an string for the calculated complexity
 std::string GetBigOString(BigO complexity);
 
+const auto StatisticsSum = [](const std::vector<double>& v) {
+  return std::accumulate(v.begin(), v.end(), double());
+};
+
+const auto StatisticsMean = [](const std::vector<double>& v) {
+  if (v.size() == 0) return double();
+  return StatisticsSum(v) * (1.0 / v.size());
+};
+
+const auto StatisticsMedian = [](const std::vector<double>& v) {
```

Comment:
why are these lambdas? it seems they could just as easily be concrete functions declared here (or in a `src/statistics.h`) and defined in an implementation file.